assets/scss/templates/_header.scss

```diff
@@ -368,15 +368,20 @@
   }
 
   .user-plan--container {
-    display: flex;
-    align-items: center;
-    border-top: 1px solid $dp-border-grey;
-    padding: 10px 0;
+    padding: 10px 0 5px;
 
     .user-plan {
       color: $dp-color-green;
       font-size: 11px;
       text-transform: uppercase;
+      margin-top: 10px;
+    }
+
+    .user-plan-enabled {
+      font-size: 11px;
```

Comment:
I think we have a variable for font sizes...

Reply:
changed